### PR TITLE
Add ability to compare selection strategies

### PR DESCRIPTION
**Pull request by i1skn**
Before tx creation user can estimate fee and locked amount
with different selection strategies by providing `-e` flag for
`wallet send` command.

### CODE REVIEW

File: src/bin/cmd/wallet_args.rs [hunk 1]
```diff
@@ -562,7 +574,11 @@ pub fn wallet_command(
 		}
 		("send", Some(args)) => {
 			let a = arg_parse!(parse_send_args(&args));
-			command::send(inst_wallet(), a)
+			command::send(
+				inst_wallet(),
+				a,
+				wallet_config.dark_background_color_scheme.unwrap_or(true),
```
Comment (hashmap) [inline src/bin/cmd/wallet_args.rs]: Is it unrelated change?

**Comment by kargakis** [comment]
Why not use `--dry-run` which is clearer than `-e`?

**Comment by i1skn** [comment]
@kargakis short version `-d` is already taken for a `--destination`. `-e` states for `--estimate-selection` which, I think, also makes sense.

**Comment by yeastplume** [comment]
Looks good for now. Might want to explore refactoring some of the underlying coin selection calls to eliminate a bit of redundancy, but can leave that until after the wallet is split off.
